src/components/treeActivity/index.tsx

```diff
@@ -35,19 +38,72 @@ const EntryMessage = styled(Paragraph)`
   color: ${TEXT_GREY};
 `;
 
+const StewardshipActivityDropdownContainer = styled.div`
+  margin: -10px 0 -5px 5px;
+`;
+
+const StewardshipActivityDropdown = styled.div`
+  font-size: 15px;
+  color: ${MID_GREEN};
+`;
+
+const CenteredPagination = styled(Pagination)`
+  display: flex;
+  justify-content: center;
+  & .ant-pagination-item-active {
+    background-color: white;
+  }
+  & .ant-pagination-item-active :hover {
+    background-color: ${DARK_GREEN};
+  }
+`;
+
 interface TreeActivityProps {
   readonly stewardship: TreeCare[];
-  readonly limit?: number;
+  readonly monthYearOptions: MonthYearOption[];
 }
 
-const TreeActivity: React.FC<TreeActivityProps> = ({ stewardship, limit }) => {
+const TreeActivity: React.FC<TreeActivityProps> = ({
+  stewardship,
+  monthYearOptions,
+}) => {
+  const [selectedMonthYear, setSelectedMonthYear] = useState(
+    new Date().toLocaleString('default', {
+      month: 'short',
+      year: 'numeric',
+    }),
+  );
+  const selectedMonthYearStewardship: TreeCare[] = stewardship.filter(
+    (entry) =>
```

Comment:
We should change the way TreeCare/entries are stored. It doesn't make much sense to convert the stewardship to  strings then do a bunch of substring stuff in the filtering. Change TreeCare to have month, day, and year in the exact format you need to filter them then convert only the filtered values to the correct string formats after.